inspirehep/modules/disambiguation/core/ml/models.py

```diff
@@ -76,3 +101,174 @@ def fit(self):
 
     def predict(self, X):
         return self.estimator.predict(X)
+
+
+class DistanceEstimator(object):
+    def __init__(self, ethnicity_model_path):
+        self.ethnicity_estimator = EthnicityEstimator()
+        self.ethnicity_estimator.load_model(ethnicity_model_path)
+
+    def load_data(self, signatures_path, pairs_path, pairs_size, publications_path):
+        reversed_publications = {}
```

Comment:
I know @jacquerie has used this terminology earlier, but it's not a priori clear in which sense this is reversed. I would call this `publications_by_id` and similarly `signatures_by_uuid`, etc.

Reply:
Good idea. Will amend the current PR and add a commit to fix the previous uses.

Reply:
This happened in https://github.com/inspirehep/inspire-next/pull/3455/commits/c9c6e5917c7412c56eeadc55d28159bb65cd977c.